packages/pwa-kit-react-sdk/src/ssr/universal/utils.server.test.js

```diff
@@ -7,7 +7,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-
+/* eslint-disable header/header */
```

Comment:
this doesn't seem right. i don't see any other changes in this file. is this intended?